integration_test/ddqn/test.py

```diff
@@ -2,16 +2,14 @@
 import sys
 import yaml
 sys.path.append('../rl-bakery/')
```

Comment:
I didn't catch that sys.path.append before, we should remove that as well. We shouldn't run it from the ddqn/ dir